dataflow/gpu-workers/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM tensorflow/tensorflow:2.3.1-gpu
+
+WORKDIR /root
+
+# Installing the requirements here makes the workers start faster since they
+# don't need to install the requirements at runtime.
+#   ℹ️ Make sure your requirements.txt includes `apache-beam[gcp]`.
+COPY requirements.txt .
+
+# Don't install `tensorflow` since the base image already comes with it.
+RUN egrep -v '^tensorflow([=<>]|$)' requirements.txt > /tmp/requirements.txt \
+    && pip install -U pip \
+    && pip install -r /tmp/requirements.txt
```

Comment:
Use the `--no-cache-dir` flag with `pip install` to slim down the image - the cache isn't needed in Docker :)